util/new_keyboard.sh

```diff
@@ -32,6 +32,8 @@ set_git_username() {
 
 # Copy the template files to the new keyboard directory.
 copy_templates() {
+    mkdir -p "$(dirname $keyboard_dir)"
+
     echo -n "Copying base template files..."
     cp -r "quantum/template/base" "${keyboard_dir}"
```

Comment:
```suggestion
    cp -r "quantum/template/base/." "${keyboard_dir}"
```

A mistake I made when rewriting this script.